extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1467,7 +1467,7 @@ export class AmpStoryPage extends AMP.BaseElement {
     const ampVideoEls = this.element.querySelectorAll('amp-video');
     return Array.prototype.some.call(
       ampVideoEls,
-      (video) => !video.hasAttribute('noaudio')
+      (video) => !video.hasAttribute('noaudio') && video.getAttribute('volume') !== '0'
```

Comment:
nit (in case we move forward with this PR, and I don't review it): We should parse this as a number and compare it to 0, so that `volume="0.0"` would behave the same way as `volume="0"`.